coreneuron/nrnoc/fadvance_core.cpp

```diff
@@ -34,6 +34,9 @@ THE POSSIBILITY OF SUCH DAMAGE.
 #include "coreneuron/nrniv/nrn_acc_manager.h"
 #include "coreneuron/utils/reports/nrnreport.h"
 #include "coreneuron/utils/progressbar/progressbar.h"
+#include "coreneuron/nrniv/profiler_interface.h"
+
+#include <sstream>
```

Comment:
standard include should be before coreneuron ones. clang-format should do it automatically.